### ISSUE TITLE
Banner - meaningful vs. decorative

### ISSUE THREAD

**Issue by katknow** [omeka-s-themes/default#44]
Currently, the banner is being pinged by the Accessibility Insights for Web assessment as meaningful. I think this should be labeled as decorative?
